doc/io_chaos.md

```diff
@@ -0,0 +1,157 @@
+# IO Chaos document
+
+This document will help user to build IO Chaos experiments. 
+
+IO Chaos can help user simulate file system faults such as I/O delay, read/write errors, etc. It can inject delay and errno when user using syscall about IO like `open`, `read`, `write`. 
+
+## Sample Config
+
+Sample IO chaos ducument:
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: IoChaos
+metadata:
+  name: io-delay-example
+  namespace: chaos-testing
+spec:
+  action: mixed
+  mode: one
+  duration: "400s"
+  configName: "chaosfs-tikv"
+  path: ""
+  selector:
+    namespaces:
+      - tidb-cluster-demo
+    labelSelectors:
+      "app.kubernetes.io/component": "tikv"
+  layer: "fs"
+  percent: "50"
+  delay: "1ms"
+  scheduler:
+    cron: "@every 10m"	
+```
+
+User can find and edit the template refer to [examples/io-mixed-sample.yaml](../examples/io-mixed-sample.yaml).
+
+## Usage
+
+### Config
+
+#### Annotations
+
+We use [Kubernetes annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) to  attach metadata about IO chaos to objects. In [examples/io-mixed-sample.yaml](../examples/io-mixed-sample.yaml), user can find metadata below.
+
+```yaml
+metadata:
+  name: io-delay-example
+  namespace: chaos-testing
+```
+
+If you not attach annotation to namespace, it will modify the pod dynamically, and may restart the pod.
+
+#### Data
+
+The data directory of the component should be a subdirectory of `PersistentVolumes`.
```

Comment:
Of what component? You mean "Chaos Operator"? @mapleFU

Reply:
I'll add it.